solidity/src/BftBridge.sol

```diff
@@ -188,16 +173,22 @@ contract BFTBridge is TokenManager, UUPSUpgradeable, OwnableUpgradeable, Pausabl
     /// Burn ERC 20 tokens there to make possible perform a mint on other side of the bridge.
     /// Caller should approve transfer in the given `from_erc20` token for the bridge contract.
     /// Returns operation ID if operation is succesfull.
-    function burn(uint256 amount, address fromERC20, bytes memory recipientID) public whenNotPaused returns (uint32) {
+    function burn(uint256 amount, address fromERC20, bytes32 toTokenID, bytes memory recipientID)
```

Comment:
Add these tests:
- burn is called with not registered token, isBase is true, the burn does not fail
- burn is called with not registered token, isBase is false, the burn fails
- mint is called with not registered token, isBase is true, the mint does not fail
- mint is called with not registered token, isBase is false, the mint fails